src/database/schema.ts

```diff
@@ -61,3 +75,74 @@ export const sessionRelations = relations(session, ({ one }) => ({
     references: [users.uuid],
   }),
 }))
+
+export const jobs = pgTable('jobs', {
+  ...createdAt,
+  uuid: uuid('uuid')
+    .default(sql`gen_random_uuid()`)
+    .primaryKey(),
+  user_uuid: uuid('user_uuid')
+    .notNull()
+    .references(() => users.uuid),
+  name: varchar('name'),
+  status: varchar('status').notNull(),
```

Comment:
you can use `text` instead of varchar

Reply:
I have read that `text` is a better option for fields where we don't know their length, so I updated `description` field in the tickets table.